internal/connect/command_cluster_create.go

```diff
@@ -25,6 +25,36 @@ func (c *clusterCommand) newCreateCommand() *cobra.Command {
 		RunE:        c.create,
 		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
 		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "The configuration file contains configs including offsets for the connector.",
+				Code: `{
+  "name": "MyGcsLogsBucketConnector",
```

Comment:
What are GCS logs? Can we update the text of this example to explain what they are?

Reply:
Its just a example of connector type.  Doesnt have any specific relation here.